cluster/gce/gci/master-helper.sh

```diff
@@ -135,6 +136,7 @@ function create-master-instance-internal() {
   metadata="${metadata},kube-master-certs=${KUBE_TEMP}/kube-master-certs.yaml"
   metadata="${metadata},cluster-location=${KUBE_TEMP}/cluster-location.txt"
   metadata="${metadata},controllerconfig=${KUBE_TEMP}/controllerconfig.json"
+  metadata="${metadata},networktemplate=${KUBE_TEMP}/network.tmpl"
```

Comment:
A condition judgment is required here because ${KUBE_TEMP}/network.tmpl is not always exist after deleting default generation

Reply:
add it.

Reply:
I do same thing for cluster/gce/gci/scaleoutserver-helper.sh.